apps/designer-standalone/src/app/AzureLogicAppsDesigner/Services/WorkflowAndArtifacts.tsx

```diff
@@ -108,9 +108,8 @@ export const listCallbackUrl = async (
   }
 
   let callbackUri: URL;
-
   if (callbackInfo.relativePath) {
-    callbackUri = new URL(callbackInfo.basePath, validateResourceId(callbackInfo.relativePath));
+    callbackUri = new URL(`${callbackInfo.basePath}${validateResourceId(callbackInfo.relativePath)}`);
     Object.entries(callbackInfo.queries).forEach(([key, value]) =>
       callbackUri.searchParams.append(encodeURIComponent(key), encodeURIComponent((value as any) ?? ''))
```

Comment:
Looks like it's double encoded here. When adding the search parameters, we should not encode it. URL.toString() will emit the correct URL by encoding the parameter